tests/System/Routes/AddCommentRouteTest.php

```diff
@@ -40,7 +40,30 @@ public function testGivenRequestWithoutParameters_resultIsError() {
 		$this->assertErrorJsonResponse( $response );
 	}
 
+	public function testGivenRequestWithoutTokens_resultIsError() {
+		// TODO: insert donation
+
+		$client = $this->createClient();
+
+		$client->request(
+			'POST',
+			'add-comment',
+			[
+				'kommentar' => 'Your programmers deserve a raise',
+				'public' => '1',
+				'eintrag' => 'Uncle Bob',
+				'sid' => '9001',
+			]
+		);
+
+		$response = $client->getResponse();
+
+		$this->assertTrue( $response->isSuccessful(), 'request is successful' );
```

Comment:
This is a mismatch with the test name and the given data. Shouldn't this be `assertFalse`?